public/styles.css

```diff
@@ -1,14 +1,34 @@
+body {
+  background-color: #E5E5E5;
+}
+
+.header {
+  position: absolute;
```

Comment:
style nit (and I know I didn't do this with the original css) please alphabetize the css values within each object. Thanks!

Reply:
To confirm,  should something like ` box-sizing: border-box;` be changed to `box-sizing: Border-Box;`?

Reply:
no, all lower case is the convention in CSS so `box-sizing: border-box;` is fine.